### ISSUE TITLE
Missing filter for users that have not yet logged in

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#10820]
## Describe the bug
relates to https://github.com/owncloud/enterprise/issues/6597

## Steps to reproduce

1. login on https://ocis.ocis-web.latest.owncloud.works/
2. Go to `Admin Settings` > `Users`
3. create a new user `Schorsch`
5. Filter by `Roles` > `User`
6. Schorsch is not listed

## Expected behavior
- Option 1: Schorsch gets listed
Alternative:
- Option 2: Add role filteroption `None (Never logged in)` that shows all users without a role. Note: This solution would be a workaround for the situation that we don't know the role until the user loggs in at least once.

![screenshot_002631](https://github.com/owncloud/web/assets/26610733/fa2c8262-4693-40f5-91a9-c630abf2488c)

## Actual behavior
- Schorsch is not listed

**Comment by tbsbdr** [comment]
@tbsbdr add backend ticket

**Comment by AlexAndBear** [comment]
This might need a server implementation (filter)

**Comment by AlexAndBear** [comment]
@rhafer can we get a filter in this sprint? It seems to miss and blocks this ticket

**Comment by rhafer** [comment]
Just for completeness: We just discussed this and figured that this needs some more thoughts as the resulting (short-term) solution would be  quite slow. Relates a bit to https://github.com/owncloud/ocis/issues/8938 . We're postponing this for now.

**Comment by AlexAndBear** [comment]
I removed it from the sprint, pr with the current process will be linked, if we decide to go that route anyways later

**Comment by dj4oC** [comment]
Before this can be acted on, the following needs to be resolved:

- The thread ends with the ticket explicitly postponed and pulled from the sprint: "this needs some more thoughts as the resulting (short-term) solution would be quite slow," relating to `owncloud/ocis#8938`.
- A required backend filter for efficient querying was identified as missing at the time, with no confirmation it has since been added.

Could someone confirm whether `owncloud/ocis#8938` (or an equivalent backend filter) has since landed, which would unblock this?

🤖 Automated by Claude Code

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
